pyproject.toml

```
@@ -0,0 +1,66 @@
+[build-system]
+requires = ["setuptools>=61.2"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "codespell"
+dynamic = ["version"]
+maintainers = [{name = "Lucas De Marchi", email = "lucas.de.marchi@gmail.com"}]
+license = {text = "GPL v2"}
+description = "Codespell"
+classifiers = [
+    "Intended Audience :: Developers",
+    "License :: OSI Approved",
+    "Programming Language :: Python",
+    "Topic :: Software Development",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX",
+    "Operating System :: Unix",
+    "Operating System :: MacOS",
+]
+requires-python = ">=3.7"
+
+[project.readme]
+file = "README.rst"
+content-type = "text/x-rst"
+
+[project.urls]
+Homepage = "https://github.com/codespell-project/codespell/"
+Download = "https://github.com/codespell-project/codespell/"
```

Should this be https://github.com/codespell-project/codespell/releases perhaps?
```suggestion
Download = "https://github.com/codespell-project/codespell/releases"
```

What do other people do?